docs/system/system-metrics.md

```diff
@@ -15,154 +15,689 @@ instruments not explicitly defined in the specification.
-| ------------------------ | --------------------------------------------------------- | ----------- | ------------------------------------------------- | ---------- | ------------- | ---------------------------------------------------------------------------------------------- |
-| system.processes.count   | Total number of processes in each state                   | {process} | UpDownCounter                                     | Int64      | status        | running, sleeping, [etc.](https://man7.org/linux/man-pages/man1/ps.1.html#PROCESS_STATE_CODES) |
-| system.processes.created | Total number of processes created over uptime of the host | {process} | Counter                                           | Int64      | -             | -                                                                                              |
-
-### `system.{os}.` - OS Specific System Metrics
+## Processor Metrics
+
+**Description:** System level processor metrics captured under the namespace `system.cpu`.
+
+### Metric: `system.cpu.time`
+
+This metric is [recommended][MetricRecommended].
+
+<!-- semconv metric.system.cpu.time(metric_table) -->
+| Name     | Instrument Type | Unit (UCUM) | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `system.cpu.time` | Counter | `s` | Seconds each logical CPU spent on each mode |
+<!-- endsemconv -->
+
+<!-- semconv metric.system.cpu.time(full) -->
+| Attribute  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| `system.cpu.cpu` | int | The logical CPU number [0..n-1] | `1` | Recommended |
+| `system.cpu.state` | string | The state of the CPU | `idle`; `interrupt` | Recommended |
+
+`system.cpu.state` has the following list of well-known values. If one of them applies, then the respective value MUST be used, otherwise a custom value MAY be used.
+
+| Value  | Description |
+|---|---|
+| `idle` | idle |
+| `user` | user |
+| `system` | system |
+| `interrupt` | interrupt |
+<!-- endsemconv -->
+
+### Metric: `system.cpu.utilization`
+
+This metric is [recommended][MetricRecommended].
+
+<!-- semconv metric.system.cpu.utilization(metric_table) -->
+| Name     | Instrument Type | Unit (UCUM) | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `system.cpu.utilization` | Gauge | `1` | Difference in system.cpu.time since the last measurement, divided by the elapsed time and number of logical CPUs |
+<!-- endsemconv -->
+
+<!-- semconv metric.system.cpu.utilization(full) -->
+| Attribute  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| `system.cpu.cpu` | int | The logical CPU number [0..n-1] | `1` | Recommended |
```

Comment:
```suggestion
| `system.cpu.logical.count` | int | The logical CPU number [0..n-1] | `1` | Recommended |
```

Reply:
Hum, from the description I thought this was like the "assigned number" of the logical CPU, not how many of them are present. So calling `logical.count` is wrong? I think `system.cpu.logical` is enough maybe?

Reply:
@frzifus I changed it to `system.cpu.logical` in [19cb403](https://github.com/open-telemetry/semantic-conventions/pull/89/commits/19cb403c98d10562214187631d2b572fa776b358). Let me know if you think this makes sense.

Reply:
@frzifus hi! Did you had a chance to look at my reply here? Does that resolve your concern?

Reply:
@joaopgrassi Is it in that case a duplicate of https://github.com/open-telemetry/semantic-conventions/pull/89/files#diff-a5f4e0108cc187e4aeaded0612532f0f3d65c0261854c66389273b57c1faa0cbR64-R78?

Reply:
> @joaopgrassi Is it in that case a duplicate of https://github.com/open-telemetry/semantic-conventions/pull/89/files#diff-a5f4e0108cc187e4aeaded0612532f0f3d65c0261854c66389273b57c1faa0cbR64-R78?

AIUI these are different: `system.cpu.logical` is a metric _attribute_ name while `system.cpu.logical.count` is a metric name

Reply:
Yeah they are different, but I can maybe see it being confusing. Maybe we can change the attribute to `system.cpu.logical_number`?

Reply:
ok, I see - thanks for clarification! I like `system.cpu.logical_number` what do you think?

Reply:
Alright, I will change it then!

Reply:
Awesome

Reply:
Done in [e7bbf62](https://github.com/open-telemetry/semantic-conventions/pull/89/commits/e7bbf62a44cbac66de1d202ce3a9a1962da792db)